image/save/interface.go

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package save
+
+import (
+	"context"
+)
+
+//SaveImage can save a list of images of the specified architecture
+type ImageSave interface {
+	SaveImages(images []string, dir, arch string) error
```

Comment:
i think parameter "arch" we can fetch from the os. user no need to care it. image arch strong related to the docker deamon while that field also from the os.

Reply:
@kakaZhou719 But how can you pull ARM image in x86 server. Fetch from the os can let the caller decide.

Reply:
platform of arm image have variant like armv7 ,armv8, i suggest we use oci platform to do this rather than sting.  "github.com/opencontainers/image-spec/specs-go/v1"